test/extended/testdata/bindata.go

```diff
@@ -4949,11 +4949,11 @@ spec:
       - -c
       - |
         set -e
-        sleep 15  # give tests time to set up
         openshift-deploy --until=50%
         echo Halfway
         openshift-deploy
         echo Finished
+        sleep 15  # give tests time to observe logs
```

Comment:
Might be better to create a separate sleep at the end so when cio gets fixed the removal is not error prone since the setup sleep is actually needed.

Reply:
I don't see why the setup sleep is needed?

Reply:
logs -f starts from the pod beginning - there used to be a bug in crio where it dropped some start logs, but that's fixed now.

Reply:
Without it the deplyoer can run too fast and logs would race with deployer controller wanting to delete that pod immediately

Reply:
That's not how this works.  This is inside the deployer pod.  The moment this pod completes, a CLI component (that waited for rollout status) is racing with the deployer pod.